model/src/main/java/org/projectnessie/model/BaseMergeTransplant.java

```diff
@@ -66,16 +66,18 @@ public interface BaseMergeTransplant {
   @JsonSerialize(as = ImmutableMergeKeyBehavior.class)
   @JsonDeserialize(as = ImmutableMergeKeyBehavior.class)
   interface MergeKeyBehavior {
+    @Value.Parameter(order = 1)
     ContentKey getKey();
 
+    @Value.Parameter(order = 2)
     MergeBehavior getMergeBehavior();
 
     static ImmutableMergeKeyBehavior.Builder builder() {
       return ImmutableMergeKeyBehavior.builder();
     }
 
     static MergeKeyBehavior of(ContentKey key, MergeBehavior mergeBehavior) {
```

Comment:
Question: Do we really need one more wrapper on top of the existing ImmutableMergeKeyBehavior.of()? why can't we directly use ImmutableMergeKeyBehavior.of() ?